AquaNet/src/pages/UserHome.svelte

```diff
@@ -172,7 +172,9 @@
             <div class="info">
               <div>{r.name ??"Unable find music"}</div>
               <div>
-                <span class={`lv level-${r.level}`}>{ r.notes?.[r.level]?.lv?.toFixed(1) ?? r.level ?? '0'}</span>
+                <span class={`lv level-${r.level === 10 ? 3 : r.level}`}>
+                  { r.notes?.[r.level === 10 ? 0 : r.level]?.lv?.toFixed(1) ?? r.level ?? '0'}

```

Comment:
Statement seems too complex.
Consider change to a self-exec func with if statement.

btw `Number(a.toFixed(1))` may better than `a.toFixed(1)`

Reply:
This is intended. Having "13.1" and "13.0" in the same column looks better than "13.1" and "13"

![image](https://github.com/hykilpikonna/AquaDX/assets/22280294/cd30f937-f981-428f-b5cb-da90dc65a911)
![image](https://github.com/hykilpikonna/AquaDX/assets/22280294/2d8111e8-90c9-45c5-98bb-bc7357b1c762)